README.md

```diff
@@ -34,24 +168,17 @@ scrape_configs:
 
   # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
   - job_name: 'maia'
-    metrics_path: "/api/v1/metrics"
+    metrics_path: "/federate"
```

Comment:
It is not part of the versioned API in Prometheus. Please also consider: That way I can use the same CLI commands also for querying Prometheus.